Formula/chisel.rb

```diff
@@ -1,15 +1,23 @@
 class Chisel < Formula
   desc "Collection of LLDB commands to assist debugging iOS apps"
   homepage "https://github.com/facebook/chisel"
-  url "https://github.com/facebook/chisel/archive/1.6.0.tar.gz"
-  sha256 "63f6538c7221e51e6133f62b3f0c0a74f84feee9727c80720da5f63a78f6db93"
+  url "https://github.com/facebook/chisel/archive/1.7.1.tar.gz"
+  sha256 "952354d358dea8407b36effff2dd6acd81af733ca920ec2a97e62235f5bcc749"
   head "https://github.com/facebook/chisel.git"
 
-  bottle :unneeded
-
   def install
     libexec.install Dir["*.py", "commands"]
     prefix.install "PATENTS"
+
+    # == LD_DYLIB_INSTALL_NAME Explanation ==
+    # This make invocation calls xcodebuild, which in turn performs ad hoc code
+    # signing. Note that ad hoc code signing does not need signing identities.
+    # Brew will update binaries to ensure their internal paths are usable, but 
```

Comment:
The brew test bot isn't liking this one particular space at the very end!  Can you remove it real quick?

Reply:
fixed, thanks.

Reply:
Perfect!  We'll let the brew test bot do it's thing now.  :wink: